.travis.yml

```diff
@@ -1,5 +1,13 @@
 language: c
-install: wget https://raw.githubusercontent.com/ocaml/ocaml-travisci-skeleton/master/.travis-opam.sh
-script: bash -ex .travis-opam.sh
+# Avoid running in a container, as this doesn't work for the ocaml tests.
+sudo: required
+python:
+  - "2.7"
+install:
+  - wget https://raw.githubusercontent.com/ocaml/ocaml-travisci-skeleton/master/.travis-opam.sh
+  - pip install --user flake8
+script:
+  - bash -ex .travis-opam.sh
```

Comment:
Great!! One last thing. Let's re-order these checks so that it fails-fast. Checking flake8 will be *much* quicker that executing the .travis-opam.sh.